src/Sulu/Bundle/MediaBundle/Resources/public/js/components/collections/masonry-decorator/masonry-view.js

```diff
@@ -0,0 +1,414 @@
+/**
+ * @class Masonryview (Datagrid Decorator)
+ * @constructor
+ *
+ * @param {Object} [viewOptions] Configuration object
+ * @param {Boolean} [unselectOnBackgroundClick] should items get deselected on document click
+ * @param {Boolean} [selectable] should items be selectable
+ * @param {String} [imageFormat] api-format which is used for head-image
+ * @param {String} [emptyListTranslations] translation key for the empty-list indicator
+ * @param {Object} [fields] Defines which data-columns are used to render cards
+ * @param {String} [fields.image]
+ * @param {Array} [fields.title]
+ * @param {Array} [fields.description]
+ * @param {Object} [separators] Defines separators between data-columns
+ * @param {String} [separators.description]
+ *
+ * @param {Boolean} [rendered] property used by the datagrid-main class
+ * @param {Function} [initialize] function which gets called once at the start of the view
+ * @param {Function} [render] function to render data
+ * @param {Function} [addRecord] function to add a new record to the grid
+ * @param {Function} [removeRecord] function to remove an existing record from the grid
+ * @param {Function} [destroy] function to destroy the view and unbind events
+ */
+define(function() {
+
+    'use strict';
+
+    var defaults = {
+            unselectOnBackgroundClick: true,
+            selectable: true,
+            imageFormat: '190x',
+            emptyListTranslation: 'public.empty-list',
```

Comment:
@marcelmoosbrugger setting it in defaults makes it possible to defin a custom emptList-text, for example "collection is empty"